README.md

```diff
@@ -79,8 +92,6 @@ jobs:
       matrix_exclude: '[{"ruby": "2.5", "node": "10"}, {"ruby": "2.5", "node": "12"}]'
 ```
 
-By default, this will run against Foreman `develop` branch.
-
 You can adjust this by adding another matrix:
```

Comment:
By dropping the line above, the "this" makes no sense anymore.

How about
```suggestion
You can run tests against multiple Foreman versions by using a matrix:
```

Reply:
Done